modules/cms/controllers/index/_safemode_notice.htm

```diff
@@ -0,0 +1,6 @@
+<div class="callout fade in callout-info no-subheader">
+    <div class="header">
+        <i class="icon-info"></i>
+        <h3><?= e(trans('cms::lang.cms_object.safe_mode_enabled')) ?></h3>
```

Comment:
Can you change the trans key to say "Safe mode is currently enabled. Editing the PHP code of CMS templates is disabled."

Reply:
Who's going to handle all the translations for the other languages?

Reply:
Whoever wants to / notices that it changes. It's pretty much the same string, just more descriptive. The way October handles translations in the core is entirely community driven, the only language we actively keep up to date (besides moving keys around) is English